ajax/libs/jodit/package.json

```diff
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
   "author": "Chupurnov <chupurnov@gmail.com> (http://xdsoft.net/)",
   "repository": {
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
```

Comment:
Why have you added git auto-update when the npm auto-update config already exists?

Reply:
Removed